# HG changeset patch
# User Rik <rik@octave.org>
# Date 1363807565 25200
#      Wed Mar 20 12:26:05 2013 -0700
# Node ID dda043ccad7cab95a72b3a6283a0cf228dad723b
# Parent  b73b5ab6c277a65bf2e51ed9cd54bcb08cdde068
__voronoi__.cc: Change QHull arguments for 4D inputs to match Matlab.

* libinterp/dldfcn/__voronoi__.cc(F__voronoi__): Use Qhull options "Qbb Qx" for
inputs >= 4D.

diff --git a/libinterp/dldfcn/__voronoi__.cc b/libinterp/dldfcn/__voronoi__.cc
--- a/libinterp/dldfcn/__voronoi__.cc
+++ b/libinterp/dldfcn/__voronoi__.cc
@@ -85,17 +85,17 @@ Undocumented internal function.\n\
   Matrix points = args(1).matrix_value ();
   const octave_idx_type dim = points.columns ();
   const octave_idx_type num_points = points.rows ();
 
   points = points.transpose ();
 
   std::string options;
 
-  if (dim <= 4)
+  if (dim <= 3)
     options = " Qbb";
   else
     options = " Qbb Qx";
 
   if (nargin == 3)
     {
       octave_value opt_arg = args(2);
 
